.codespellrc

```diff
@@ -0,0 +1,3 @@
+[codespell]
+skip = ./testing/**/*,./.git/**/*,./application/cxxopts.hpp,./cmake/NSIS.template.in
+ignore-words-list=nnumber,unknwn,dota
```

Comment:
Why are we ignoring these words?

Reply:
a few false positive, 

`cmake/testing.cmake:f3d_test(NAME TestVerbose DATA dragon.vtu ARGS -s --verbose REGEXP "Number of points: 69827\nNumber of cells: 139650" NO_RENDER)`

`winshellext/F3DShellExtensionClassFactory.h:#include <unknwn.h>`

`doc/GALLERY.md:*Animated, skinned and morphed rendering*: f3d dota/scene.gltf `